### PR TITLE
Removed loading animations

### PR DESCRIPTION
**Pull request by MarsBarLee**
Removed loading icon and fade-in animation before each page loads.

For the front-page, there is a if-statement for the fade-in animation. It's currently set to false.

[Issue 101](https://github.com/numpy/numpy.org/issues/101)

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -6,28 +6,26 @@
...
     <!-- Back To Top Button -->
     <div id="backtotop"><a href="#" id="backtotop-color"></a></div>
 
-    {{ if .Site.Params.sidebar }}
-    {{ partial "sidebar.html" . }}
-    {{ end }}
-
-    {{ partial "javascript.html" . }}
+    {{ if .Site.Params.sidebar }} {{ partial "sidebar.html" . }} {{ end }} {{
+    partial "javascript.html" . }}
```
Comment (joelachance) [inline layouts/_default/baseof.html]: IMO, I have a harder time reading this as a single line. I think nesting and using multiple lines is just fine here!
Same on LOC 9, 13, & 23.

**Comment by rgommers** [comment]
@MarsBarLee this doesn't actually build for me, it fails with:
```
$ hugo
Total in 10 ms
Error: add site dependencies: load resources: loading templates: 
"/home/rgommers/code/websites/numpy.org/layouts/_default/baseof.html:28:1": parse master failed: 
template: index.html:28: unexpected unclosed action in command
```

If it does work for you, then it's probably due to using different Hugo versions. Mine is:
```
$ hugo version
Hugo Static Site Generator v0.62.0/extended linux/amd64 BuildDate: unknown
```

File: layouts/_default/baseof.html (hunk 1 above)
Comment (rgommers) [reply layouts/_default/baseof.html]: The line break at the end of line 28 probably causes my build failure. Undoing this change should fix it.

Comment (MarsBarLee) [reply layouts/_default/baseof.html]: I realized that this problem is caused by my code editor and formatting. It moves the lines and white text around when I save. I'll change my settings and update my pull request without these problems.

**Comment by joelachance** [comment]
This one loads for me. However, I did notice there's some 'flash' when the page first loads since we're still fetching fonts, etc. We can avoid this by doing this:

`{{ if .IsHome }}` instead of `{{ if false }}`.
source: https://discourse.gohugo.io/t/how-to-check-for-home-page/149/24

When trying that, I don't see the loading icon, but it does seem to fade in nicely. No loader on the other pages.

Otherwise looks great!

**Comment by MarsBarLee** [comment]
> This one loads for me. However, I did notice there's some 'flash' when the page first loads since we're still fetching fonts, etc. We can avoid this by doing this:
> 
> `{{ if .IsHome }}` instead of `{{ if false }}`.
> source: https://discourse.gohugo.io/t/how-to-check-for-home-page/149/24
> 
> When trying that, I don't see the loading icon, but it does seem to fade in nicely. No loader on the other pages.
> 
> Otherwise looks great!

Ok, made those changes and pushed! If we decide to remove the loading entirely, then we'll remove the preloader section altogether.

**Comment by rgommers** [comment]
First PR merged! Nice @MarsBarLee :)

**Pull request merged**